demisto_sdk/__main__.py

```diff
@@ -280,28 +283,29 @@ def unify(**kwargs):
 )
 @click.option('-i', '--input',
               help=("The packs to create artifacts for. Optional values are: `all` or "
-                    "csv list of pack names. "))
+                    "csv list of pack names. "), required=True)
```

Comment:
I think that you should use the `PathsParamType` here: https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/__main__.py#L71

Reply:
In that implementation - the entire command will fail even if i have mistake in one pack path
i implement check that just ignore the invalid path and do the work on the valid packs
should i change it?

Reply:
@roysagi Your input here is needed - what is the way we want to go here?

Reply:
Done